### PR TITLE
Use format.person method to retrieve the name and email

### PR DESCRIPTION
**Pull request by jimhester**
This handles cases with multiple given or family names better that the
previous approach

### CODE REVIEW

File: R/email.R [hunk 1]
```diff
@@ -36,9 +36,11 @@ print.maintainers <- function(x, ...) {
 #'   failed to send in the previous round. If omitted uses all packages.
 #' @param data Optionally, supply a named list to provide your own parameters
 #'   to fill in the template
+#' @param draft If `TRUE`, create a gmail draft rather than sending the email
+#'   directly.
```
Comment (gaborcsardi) [inline R/email.R]: Can you run `document()` for this?

File: R/db.R [hunk 2]
```diff
@@ -261,5 +261,6 @@ db_results <- function(pkg, revdeps) {
 }
 
 db_maintainers <- function(pkg) {
-  dbGetQuery(db(pkg), "SELECT DISTINCT maintainer FROM revdeps")
+  res <- dbGetQuery(db(pkg), "SELECT DISTINCT maintainer, package FROM revdeps")
+  stats::setNames(res$maintainer, res$package)
```
[the remark this answers is not in the document]
Comment (jimhester) [reply R/db.R]: 👍 Why this function is in stats is beyond me :)

File: R/email.R (hunk 1 above)
Comment (jimhester) [reply R/email.R]: Done

**Comment by gaborcsardi** [comment]
Thanks!

**Pull request merged**